app/models/extension.rb

```diff
@@ -0,0 +1,16 @@
+# This "model" is not backed by the database. Its main purpose is to
+# setup and provide methods to interact with Extensions.
+
+Extension = Struct.new(:name, :description, :feature_flag_name) do
```

Comment:
What are your thoughts on adding the `keyword_init: true` parameter?  This would mean we initialize by keywords and would not rely on parameter order.   (e.g. `Extension.new(name: "Listings", description: "Longer Text", feature_flag_name: "listing_feature")`)

For myself, I prefer named keywords as it helps me read the code where we use the struct.  It also provides insight into what the method names will be.

Thoughts?

Reply:
I did consider using the named keywords initially but concluded that they weren't necessary as we're setting up a static like data structure in the extension controller and the code was going to turn out a bit clunky when we build out that array with the named parameters.

However, I'm not strongly opposed to it either.